dll/cpl/sysdm/virtmem.c

```diff
@@ -581,15 +582,13 @@ OnSelChange(HWND hwndDlg, PVIRTMEM pVirtMem)
         }
 
         /* Set current pagefile size */
-#if 0
         PageFileSizeMb = 0;
-        for (i = 0; i < 26; i++)
+        for (i = 0; i < pVirtMem->Count; i++)
         {
             PageFileSizeMb += pVirtMem->Pagefile[i].InitialSize;
```

Comment:
InitialSize attribute doesn't seem to exist, see https://travis-ci.org/github/reactos/reactos/builds/676968535#L643. Also checking your changes don't break the build (and work as excepted) is a good idea in general.

Reply:
I may finally remove this part of the evolution and open a later PR on the "Current pagefile size" part of this PR. Yes, i'll work on this tomorrow.